### ISSUE TITLE
Metric scale is underestimated by ~3

### ISSUE THREAD

**Issue by Dawars** [facebookresearch/map-anything#81]
As the images show the scale of the brandenburg gate (and many other buildings) are underestimated by ~3.
I find this surprising, what kinds of images is it expected to work well with?

<img width="830" height="685" alt="Image" src="https://github.com/user-attachments/assets/fb7abfd0-eea4-494c-be77-51c4ddceb94e" />

<img width="899" height="753" alt="Image" src="https://github.com/user-attachments/assets/66d6be6f-c072-4f0d-a31e-ec36b3157a25" />

**Comment by Nik-V9** [comment]
Hi, thanks for testing it out!

Metric scale estimation is strongly driven by priors and training data used. It also depends on how good the focal length estimation is. I believe this is a scene featured in MegaDepth where the dataset is up to scale so I wouldn't expect MapAnything to be precise.

I believe that this can be easily mitigated by finetuning on AerialMegaDepth: https://github.com/facebookresearch/map-anything/pull/36 (unfortunately we can't add any new datasets to the current release and any new datasets will be used in upcoming variants of MapAnything).

**Comment by Dawars** [comment]
I also tried with intrinsic/extrinsic priors from the colmap reconstruction but same result

**Comment by Nik-V9** [comment]
Unfortunately, I believe that only metric poses would improve the result.

**Comment by Nik-V9** [comment]
Closing issue, please reopen if further assistance is needed - will be addressed in follow up models.
